src/openzaak/components/documenten/api/serializers.py

```diff
@@ -141,7 +141,9 @@ class EnkelvoudigInformatieObjectHyperlinkedRelatedField(LengthHyperlinkedRelate
     """
 
     def get_url(self, obj, view_name, request, format):
-        obj_latest_version = obj.latest_version
+        obj_latest_version = obj.get_latest_version(self.parent)
+        if obj_latest_version is None:
+            return None
```

Comment:
Hmm, when is this the case? I didn't see a test for this or did I miss it?

Reply:
This is needed when the parent of `EnkelvoudigInformatieObjectHyperlinkedRelatedField` is a serializer and the parent.instance is a queryset, so it doesn't have the `get_informatieobject` method. 
For example, test `openzaak.components.documenten.tests.test_auth_cmis.ExternalInformatieobjecttypeScopeTests.test_oio_list` fails if that snippet is removed.